### ISSUE TITLE
Searchable not working

### ISSUE THREAD

**Issue by yeomann** [saisandeepvaddi/ant-table-extensions#60]
Hi after adding searchable as table props
whole table stops working 
![image](https://user-images.githubusercontent.com/5970751/95224006-e2070600-0802-11eb-90cd-2d64ac1ce0da.png)

**Comment by saisandeepvaddi** [comment]
Can you please create a sample codesandbox or give me the sample dataSource where I can reproduce this issue?

**Comment by whjvdijk** [comment]
Im having the exact same issue. Would like to help you get this solved. What kind of information do you need from mee?

**Comment by saisandeepvaddi** [comment]
A reproducible code or a codesandbox example with this issue or the
datasource you are using would be great help to debug.

**Comment by whjvdijk** [comment]
See attached file. 
This is a direct copy of the json from my Strapi API. My React AntD table shows the primary information of the relation (name, city, phonenumber). De rest op de data gets send so seperate components so it is needed in the output.

[data.json.txt](https://github.com/saisandeepvaddi/ant-table-extensions/files/5391090/data.json.txt)

**Comment by saisandeepvaddi** [comment]
Awesome.

I'll try to fix it tonight or this weekend. Will update the ticket soon.

**Comment by whjvdijk** [comment]
Cool! If you need any more help / info, plz let me know.

**Comment by saisandeepvaddi** [comment]
I've published v1.1.5.

Can you please update and see if it fixes the issue.

**Comment by whjvdijk** [comment]
That fixed it for me, thnx!

**Comment by whjvdijk** [comment]
Hi. It's me again :) Sorry to bother you again but i'm having some more issues with the searchable functionality.
This time the fuzzyProps don't appear to have any impact. Here is my table component:

    <Table
      dataSource={relations}
      columns={columns}
      rowKey="_id"
      size="small"
      searchableProps={{
        inputProps: {
          placeholder: "Search",
          prefix: <SearchOutlined />,
        },
        fuzzySearch: true,
        fuzzyProps: {
          treshold: 0.0,
          distance: 0,
          isCaseSensitive: true
        }
      }}
      scroll={{ y: 400 }}
    />

I'm using the same dataset as before. But it doesnt mather if i search for 'Another', 'another', 'test', 'tst' i always get multiple results where i want to make the fuzzy search a bit less fuzzy and case sensitive.

**Comment by whjvdijk** [comment]
Hi, sorry to bother you again. But im having another issue with the searchable functionality. This is my table component:

    <Table
      dataSource={relations}
      columns={columns}
      rowKey="_id"
      size="small"
      searchableProps={{
        fuzzySearch: true,
        inputProps: {
          placeholder: "Search",
          prefix: <SearchOutlined />,
        },
        fuzzyProps: {
          treshold: 0.0,
          distance: 0,
          isCaseSensitive: true
        }
      }}
      scroll={{ y: 400 }}
    />

Im using the same data as before. It doesn't matter if I search for 'Another', 'another', 'Test', 'tst'. I always get multiple results where im trying to get the fuzzy search a bit less fuzzy and case sensitive. Somehow the fuzzyProps doesn't appear to do anything.

**Comment by saisandeepvaddi** [comment]
I don't think there is any fuzzyProps prop. Did you mean fuseProps?

**Comment by whjvdijk** [comment]
You are right. So i changed it to fuseProps. But no effect. isCaseSensive has no effect. 

![Screenshot 2020-10-26 at 15 16 33](https://user-images.githubusercontent.com/51088291/97183586-4122bf80-179e-11eb-987f-f4ce98f2c565.png)

And even with a treshold of 0.0 i still don't get exact resultst. I don't mind a bit fuzzy but with a setting of 0.0 it should to better than this. Any idea's?

**Comment by saisandeepvaddi** [comment]
Can you please update to 1.1.7 and try.

Also I guess `treshold` is a typo for `threshold` as well.

**Comment by whjvdijk** [comment]
Dit the update, changed the typo. But no effect. My Props:

        fuseProps: {
          threshold: 0.0,
          isCaseSensitive: true
        }

And then the result while searching:

![Screenshot 2020-10-27 at 08 06 51](https://user-images.githubusercontent.com/51088291/97268007-7f19f500-182b-11eb-93cc-78843271da00.png)

I would expect 0.0 to be an exact match and with isCaseSensitive i shouldnt al least see the first result.

**Comment by rhidwan** [comment]
I was also facing the exact same issue. Then I updated to 1.1.7 and then the previous problem resolved but I ran into another problem. I think the problem is with nested data. Not sure though. 

Here is the sandbox link: [Sandbox Link](https://codesandbox.io/s/nervous-wave-n9ckw?fontsize=14&hidenavigation=1&theme=dark)

<img width="818" alt="Screenshot 2020-10-28 at 17 16 25GMT+6" src="https://user-images.githubusercontent.com/21958999/97429328-79aece80-1941-11eb-8443-731e3160cf56.png">
<img width="1144" alt="Screenshot 2020-10-28 at 17 16 02GMT+6" src="https://user-images.githubusercontent.com/21958999/97429351-83383680-1941-11eb-9259-8fb7727db25d.png">


Can you please take a look at this?

**Comment by saisandeepvaddi** [comment]
@rhidwan You might have checked the old documentation. 
`<SearchTableInput` needs columns too because it depends on the columns to create search keys. I forgot that in the original docs. My bad. But I added it in later versions though.
`https://ant-table-extensions.vercel.app/?path=/docs/demos--advanced-search-input` Please check the latest docs.
